src/util/util.spec.ts

```diff
@@ -72,3 +72,32 @@ describe('isError guard function', () => {
     })
   })
 })
+
+describe('escapeSparqlStr', () => {
```

Comment:
Great to have tests. :+1:

However, they don't cover certain cases. For example, the tests assert the escape of one single character, but not several characters in a string (which is important to ensure). On the other hand, it does not proove that certain characters that might be considered special are not escaped, which is intended to be compiant with the w3c spec.

So I'd suggest extending the test case coverage

Reply:
Didn't think of these cases, thanks! They were addressed in [3bcb99c](https://github.com/okp4/dataverse-portal/pull/371/commits/3bcb99cf3ea1e8722f117175071573cea29bfb29).